checker/src/main/java/org/checkerframework/checker/mustcall/MustCallTransfer.java

```diff
@@ -270,12 +270,15 @@ public void updateStoreWithTempVar(TransferResult<CFValue, CFStore> result, Node
   }
 
   /** A unique identifier counter for node names. */
-  protected long uid = 0;
+  protected static long uid = 0;
```

Comment:
This opens us up to data races.  Unlikely to cause a bug, but if it ever did that would be a nightmare to debug.  Can we use an `AtomicInteger` and `getAndIncrement()`?

Reply:
I'll also say that I would prefer this counter somehow be stored in an object scoped to the current compilation, so that concurrent builds aren't impacting each other's temporary variable numbers.  But I don't immediately know of an easy way to do that.  If we have access to the javac `Context` object somehow, we could stick the state there.  But if that's not easy, we can go with the current approach.

Reply:
I don't think there's an easy, safe way to get to the `Context` (but maybe you know one): I can get the `ProcessingEnvironment` from the checker, but I'd need to cast that to `JavacProcessingEnvironment` and then call `getContext()`, which has a specific comment in its javadoc telling you not to do that.

Reply:
Ok, that's fine.  I don't have another suggestion.